dev-guide/host-port-registry.md

```diff
@@ -103,6 +103,8 @@ Ports are assumed to be used on all nodes in all clusters unless otherwise speci
 | 9642  | ovn-kubernetes southd | sdn | 4.3 | control plane only, ovn-kubernetes only |
 | 9643  | ovn-kubernetes northd | sdn | 4.3 | control plane only, ovn-kubernetes only |
 | 9644  | ovn-kubernetes southd | sdn | 4.3 | control plane only, ovn-kubernetes only |
+| 9740  | node-observability-agent | node | 4.12 | Add-on operator (off OCP payload), localhost bound. HTTP endpoint for the node observability API |
+| 9743  | node-observability-agent | node | 4.12 | Add-on operator (off OCP payload). HTTPS endpoint for the node observability API, kube-rbac-proxy protected |
```

Comment:
The localhost-only port should go in the "Localhost-only" section below. But also, I'd suggest you use the same pattern as the sdn/ovn/etc pods, where the localhost port is 20000 more than the public port. (Because if you bind localhost 9740 then that means no one else can use it for non-localhost anyway really, so you're effectively using up one of the exposed-hostport-range ports.)